extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -885,7 +885,7 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 					name: 'python3'
 				}
 			},
-			nbformat: 4,
+			nbformat: constants.NBFORMAT,
 			nbformat_minor: 5
```

Comment:
Why is this a different minor version? And might as well make this a constant too so we're consistent.

Reply:
@lucyzhang929 I see you added this line originally. Is there any particular reason why we have this minor version number here, or was it just a copy-paste type of thing and we're free to change it?

Reply:
No particular reason - I think this was copied from other notebook tests like this one: https://github.com/microsoft/azuredatastudio/blob/14904bb671656bc9593a5c2858267f38a745f4ee/src/sql/workbench/contrib/notebook/test/electron-browser/notebookModel.test.ts#L41.